Validation/RecoTrack/python/plotting/trackingPlots.py

```diff
@@ -214,8 +214,9 @@ def _makeMVAPlots(num, hp=False):
 )
 _effandfakeDeltaRPU = PlotGroup("effandfakeDeltaRPU",
                                 _makeEffFakeDupPlots("dr"     , "#DeltaR", effopts=dict(xtitle="TP min #DeltaR"), fakeopts=dict(xtitle="track min #DeltaR"), common=dict(xlog=True)) +
+                                _makeEffFakeDupPlots("drj"     , "#DeltaR(track, jet)", common=dict(xtitle="#DeltaR(track, jet)", xlog=True))+
```

Comment:
```suggestion
                                _makeEffFakeDupPlots("drj"     , "#DeltaR(track, jet)", effopts=dict(xtitle="#DeltaR(TP, jet)"), fakeopts=dict(xtitle="#DeltaR(track, jet)"), common=dict(xlog=True))+
```
would be more correct for efficiency plot